drivers/flash/spi_nor.c

```diff
@@ -1139,20 +1132,12 @@ static int spi_nor_configure(const struct device *dev)
 	int rc;
 
 	/* Validate bus and CS is ready */
-	if (!spi_is_ready_dt(&cfg->spi)) {
+	if (!spi_is_ready(&cfg->spi)) {
```

Comment:
Why the change?

Reply:
Started from old driver file, careless mistake, fixed it.